code/game/turfs/space/space.dm

```diff
@@ -15,11 +15,6 @@
 	can_border_transition = 1
 	var/static/list/parallax_appearances
 
-/turf/space/New()
```

Comment:
I think removing this will make the server startup take a lot longer due to 500x500x6x(3?) more proc calls, some of which actually might do work inside them. What's the reason again? I'm not sure what you mean by this

> Because space wasn't inheriting from the base atoms for reasons like not wanting a million tiles to make 4 more events, beams weren't working properly.

Reply:
Because it wasn't supercalling, it wasn't inheriting the event setups from atom/New(), of which beams depend upon the one (density change)

This does mean 4 more instances per space turf, which worries me

Reply:
I think we should just create events lazily with INVOKE_EVENT, to be honest

Reply:
Could do that, yeah

Reply:
Don't really fancy doing it in this PR though

Reply:
The time is quite significant.
Bleeding-Edge:
36s95
36s30
35s54
This PR:
52s07
51s91
52s12
It annoys me because I spent several nights specifically making it faster.

Reply:
Okay, I'll revert this bit then